model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -41,6 +41,11 @@ data class OrtConfiguration(
      */
     val analyzer: AnalyzerConfiguration = AnalyzerConfiguration(),
 
+    /**
+     * The configuration of the downloader.
+     */
+    val downloader: DownloaderConfiguration? = null,
```

Comment:
Since 9196e9c8b41a336e7da25e69eff196196ca92ed4 we aim for the properties in `OrtConfiguration` to be non-nullable, so please also do so here.